jack-core/src/com/rapleaf/jack/transaction/DbMetricsImpl.java

```diff
@@ -0,0 +1,157 @@
+package com.rapleaf.jack.transaction;
+
+import com.google.common.base.Stopwatch;
+import org.apache.commons.pool2.impl.GenericObjectPool;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DbMetricsImpl implements DbMetrics {
+
+
+  private Stopwatch lifeTimeStopwatch;
+  private long lastUpdateTime;
+  private int currentConnectionsNumber;
```

Comment:
Some fields has `number` suffix and some don't, even though they are all numbers. Can you unify the naming? To make it consistent and concise, could probably drop this suffix.